lib/File/stat.pm

```diff
@@ -1,38 +1,32 @@
 package File::stat;
-use 5.006;
+use v5.38;
+no feature 'signatures'; # we use prototypes
 
-use strict;
-use warnings;
 use warnings::register;
 use Carp;
 use constant _IS_CYGWIN => $^O eq "cygwin";
 
 BEGIN { *warnif = \&warnings::warnif }
 
-our(@EXPORT, @EXPORT_OK, %EXPORT_TAGS);
+our $VERSION = '1.14';
```

Comment:
Might as well move to this to the package line if we're going modern.

Reply:
Good point, done.